bobtemplates/plone_addon/src/+package.namespace+/+package.name+/tests/test_setup.py.bob

```diff
@@ -26,7 +26,7 @@ class TestSetup(unittest.TestCase):
         from {{{ package.dottedname }}}.interfaces import (
             I{{{ package.browserlayer }}})
         from plone.browserlayer import utils
-        self.assertIn(I{{{ package.browserlayer }}}, utils.registered_layers())
+        self.assertIn(I{{{ package.browserlayer }}}, utils.registered_layers())  #noqa
```

Comment:
*Totally opinionated*: if the problem is that the line can be too long then I would rather change it to:

```
self.assertIn(
    I{{{ package.browserlayer }}},
    utils.registered_layers(),
)
```

I would be very wary to add ``# noqa`` all over the place, because it defeats the purpose of adding QA in the first place :-)